test/lib/pgBackRestTest/Common/Host/HostDbCommonTest.pm

```diff
@@ -202,17 +194,11 @@ sub configRecovery
 
     if (@stryRecoveryOption)
     {
-        $oConfig{$strStanza}{&OPTION_RESTORE_RECOVERY_OPTION} = \@stryRecoveryOption;
+        $oConfig->{$strStanza}{&OPTION_RESTORE_RECOVERY_OPTION} = \@stryRecoveryOption;
     }
 
     # Save db config file
-    iniSave($self->backrestConfig(), \%oConfig, true);
-
-    # Save backup config file
-    if ($oHostBackup->nameTest(HOST_BACKUP))
-    {
-        iniSave($oHostBackup->backrestConfig(), \%oRemoteConfig, true);
```

Comment:
yeah - this was a copy-paste error.  The backup config is never changed in this function, so it was just re-saving the file unchanged.